dangerfile.js

```diff
@@ -5,7 +5,7 @@ const tslint = require('.').default;
 const path = require('path');
 async function main() {
 	await Promise.all([
-		tslint({ configurationPath: path.join(__dirname, 'tslint.json' )})
+		tslint()
 	]);
```

Comment:
<!--
  9 failure:  " should be ' lin..., " should be ' lin..., " should be ' lin..., " should be ' lin..., Unnecessarily quo..., Unnecessarily quo..., Unnecessarily quo..., tab indentation e..., tab indentation e...
  1 warning:  Missing semicolon...
  
  
  DangerID: danger-id-Danger;
  File: dangerfile.js;
  Line: 9;
-->
- :no_entry_sign: " should be ' line - package.json (quotemark)
- :no_entry_sign: " should be ' line - package.json (quotemark)
- :no_entry_sign: " should be ' line - tasks.json (quotemark)
- :no_entry_sign: " should be ' line - tslint-with-errors.json (quotemark)
- :no_entry_sign: Unnecessarily quoted property 'isBackground' found. line - tasks.json (object-literal-key-quotes)
- :no_entry_sign: Unnecessarily quoted property 'jsRules' found. line - tslint-with-errors.json (object-literal-key-quotes)
- :no_entry_sign: Unnecessarily quoted property 'test' found. line - package.json (object-literal-key-quotes)
- :no_entry_sign: tab indentation expected line - package.json (indent)
- :no_entry_sign: tab indentation expected line - tasks.json (indent)
- :warning: Missing semicolon line - index.test.ts (semicolon)